jetbrains-core/src/software/aws/toolkits/jetbrains/core/credentials/CredentialStatusNotification.kt

```diff
@@ -28,8 +28,7 @@ class CredentialStatusNotification(private val project: Project) : ConnectionSet
                         message,
                         newState.displayMessage
                     ),
-                    createNotificationExpiringAction(actionManager.getAction("aws.settings.upsertCredentials")),
-                    createNotificationExpiringAction(RefreshConnectionAction(message("settings.retry")))
+                    *newState.actions.map { createNotificationExpiringAction(it) }.toTypedArray()
```

Comment:
Is this asterisk here intentional?

Reply:
yes this is how you unpack an array into varargs in most languages